files/fluentd/run.sh

```diff
@@ -92,4 +92,9 @@ if type -p jemalloc-config > /dev/null 2>&1 ; then
     export LD_PRELOAD=$( jemalloc-config --libdir )/libjemalloc.so.$( jemalloc-config --revision )
     export LD_BIND_NOW=1 # workaround for https://bugzilla.redhat.com/show_bug.cgi?id=1544815
 fi
+if [ -f /var/log/openshift-apiserver/audit.log.pos ] ; then
+  #https://bugzilla.redhat.com/show_bug.cgi?id=1867687
+  mv /var/log/openshift-apiserver/audit.log.pos /var/log/oauth-apiserver.audit.log
```

Comment:
If we move the pos file, don't we have to move the logs as well? Or conversely if we don't move the logs, shouldn't fluetnd start fresh with the oath logs, which it hasn't seen before?

Reply:
+1 to the second part. not sure its necessary to move the `.pos` file because it would be based on the logs its reading. if anything leaving it would allow fluentd to finish reading any `audit.log` files that are left on the node

Reply:
looking at the fluentd config we'll be reading the old logs still in addition to the new log path. however if we don't move the `.pos` file fluentd we reread the old files and potentially give us duplicate log entries

Reply:
Got it.